configuration/nginx/http.conf

```diff
@@ -24,6 +26,18 @@ server {
         client_max_body_size        20m;
     }
 
+    # rate limiting for django admin panel
+    location ^~/admin {
```

Comment:
just to check if you did right, reason behind the usage of ^~?

have you tried this manually?

Reply:
❤️